### PR TITLE
starter/portfolio-story

### PR DESCRIPTION
**Pull request by Edmund1645**
This PR adds a new starter template to the collection.

### CODE REVIEW

File: contributors/contributors.yaml [hunk 1]
```diff
@@ -1,120 +1,126 @@
 - id: tommyvedvik
   name: Tommy Vedvik
-  bio: "Co-creator of @gridsome. UX & Design."
```
Comment (mtlynch) [inline contributors/contributors.yaml:3]: This is introducing a lot of unrelated churn. Can we stick with the double quotes throughout?

File: contributors/contributors.yaml [hunk 2]
```diff
@@ -1,120 +1,126 @@
...
 - id: noxify
   name: Marcus Reinhardt
-  bio: "Developer"
+  bio: 'Developer'
   avatar: ./images/noxify.jpg
+
+- id: edmundekott
+  name: Edmund Ekott
+  bio: 'Frontend Engineer'
+  avatar: ./images/edmund.jpg
```
Comment (mtlynch) [inline contributors/contributors.yaml:125]: Can the avatar filename be either `edmundekott.jpg` or `Edmud1645.jpg`?

File: contributors/contributors.yaml (hunk 1 above)
Comment (Edmund1645) [reply contributors/contributors.yaml:3]: oh wow, thanks for pointing that out. That was my prettier config.

**Comment by Edmund1645** [comment]
Hello @mtlynch thanks to attending to my PR, the requested changes have been made, kindly review again. Thanks.

**Comment by mtlynch** [comment]
Awesome, great starter!

**Pull request merged**